client/src/Container/App/App.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import style from './App.module.scss';
+import AppHeader from '../../Component/AppHeader/AppHeader';
 
 const App = () => {
   return (
     <div className={style.App}>
```

Comment:
You have removed the `.App` class from `./App.module.scss`, so please remove style.App

Reply:
Done